layouts/shortcodes/EOL_manage.md

```diff
@@ -1,9 +1,13 @@
 <div class="admonition-warning"><p class="admonition-warning-title">Warning</p><div class="admonition-warning-text">
 
-Chef Manage is
-[deprecated](/versions.html#deprecated-products-and-versions) and does
-not work with Chef Automate 2.0. See our
-[documentation](https://automate.chef.io/docs/quickstart/) to learn more
-about Chef Automate 2. This document is no longer maintained.
+Chef Manage is [deprecated](/versions/#deprecated-products-and-versions) and is
+not supported with the Chef Infra Server and Automate 2 deployment method. Chef
```

Comment:
```suggestion
not supported with the Chef Infra Server and Automate 2 deployment method. Chef
```
Do you mean the combined Chef Infra Server and Automate 2 deployment method, or do you mean both the Chef Infra Server and Automate 2 deployment methods?

Reply:
Maybe ... and does not work with our current Chef Infra Server or Automate 2. ?

Reply:
The combined method.